RUFAS/biophysical/feed_storage/feed_manager.py

```diff
@@ -228,17 +231,18 @@ def manage_daily_feed_request(self, requested_feed: RequestedFeed, time: RufasTi
         feeds_to_remove_from_inventory = {id: 0.0 for id in requested_feed.requested_feed.keys()}
         feeds_to_purchase = {id: 0.0 for id in requested_feed.requested_feed.keys()}
         for feed_id, amount_requested in requested_feed.requested_feed.items():
-            is_fulfillable_with_inventory: bool = amount_requested <= current_feed_totals[feed_id]
-            tolerance: float = 1e-6
-            is_fulfillable_with_purchase: bool = (
-                amount_requested - current_feed_totals[feed_id]
+            available_amount = current_feed_totals[feed_id]
+            tolerance = 1e-6
+            is_fulfillable_with_inventory = amount_requested <= available_amount
+            is_fulfillable_with_purchase = (
+                amount_requested - available_amount
             ) <= self.runtime_purchase_allowance.allowances[feed_id] + tolerance
             is_request_unfulfillable = not is_fulfillable_with_inventory and not is_fulfillable_with_purchase
             if is_request_unfulfillable:
                 return False
-            feeds_to_remove_from_inventory[feed_id] = amount_requested
+            feeds_to_remove_from_inventory[feed_id] = min(amount_requested, current_feed_totals[feed_id])
```

Comment:
```suggestion
            feeds_to_remove_from_inventory[feed_id] = min(amount_requested, available_amount)
```
They are essentially the same, `available_amount = current_feed_totals[feed_id]` in line 234. Just to avoid confusion.